sample/SipInterconnect/app.js

```diff
@@ -0,0 +1,33 @@
+var express = require('express');
+var path = require('path');
+var bodyParser = require('body-parser');
+
+var routes = require('./routes/index');
+
+var app = express();
+
+// view engine setup
+app.set('views', path.join(__dirname, 'views'));
+app.set('view engine', 'ejs');
+
+// uncomment after placing your favicon in /public
+//app.use(favicon(__dirname + '/public/favicon.ico'));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(express.static(path.join(__dirname, 'public')));
+
+app.use('/', routes);
```

Comment:
i dont think there's any value in separating this code into another file for the sample, just makes the reader have to jump around.